onboarding.adoc

```diff
@@ -46,7 +45,33 @@ No two open source communities are the same.
 Even for seasoned open source contributors, when they join a new community they will need to get familiar with new tools, new processes, and even new terminologies.
 Depending on the community, seemingly simple terms like contributions, community members, projects, upstream, etc.
 may mean something slightly different and can easily cause confusion for new community members.
-  
+
+== Onboarding basics
+
+As you work to make your project's onboarding process and experience most effective, ask yourself these questions:
+
+- What can contributors add to the project?
+- Where do contributors add to the project?
+- How do contributors add to the project?
+
+Answering the first question about your project's onboarding process requires a good deal of work work.
+Incoming contributors need to know what would be the most helpful contributions they could make.
+Quite a few projects have set up tagging on their source code repositories for "first-time contributors" or something similar, which is an excellent way to direct incoming contributors to solving issues and bugs while also introducing newcomers to the source material of the project.
+
+The second question—regarding "where to contribute"—is one that projects don't pose as often as you'd think.
```

Comment:
It sounds like there's too much emphasis on code (esp. with `Git-something-or-other`). Rather than only mentioning repos, it'd be good to list other areas (e.g. forums, meetups, wiki's, etc.) where people can contribute.

Reply:
Tried a revision. Thoughts?

Reply:
👍